docs/tutorial/06-develop.md

```diff
@@ -126,7 +126,7 @@ use cosmwasm_std::Decimal;
 pub struct InstantiateMsg {  
     pub contract_name: String,  
     pub purchase_denom: String,  
-    pub merchant_address: String,  
```

Comment:
we actually do want this to be a `String` as `Addr` notes that it should not be used from deserialization from user input since it needs to be validated

Reply:
Alright, I'll fix it and resubmit.